src/models/User.ts

```
@@ -0,0 +1,36 @@
+import GenericModel from "./GenericModel"
+
+export default class User extends GenericModel {
+
+    static TABLE_NAME = "members"
+    static DATABASE_MAP: Object = {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)